testing/src/test/java/org/motechproject/nms/testing/it/api/OpsControllerBundleIT.java

```diff
@@ -368,4 +406,40 @@ private void initializeLocationData() {
         transactionManager.commit(status);
     }
 
+    // create subscriber with many subscriptions helper
+    private void createSubscriberHelper() {
+
+        TransactionStatus status = transactionManager.getTransaction(new DefaultTransactionDefinition());
+        // create subscription for a msisdn
+        rh = new RegionHelper(languageDataService, languageService, circleDataService, stateDataService,
+                districtDataService, districtService);
+
+        sh = new SubscriptionHelper(subscriptionService, subscriberDataService, subscriptionPackDataService,
+                languageDataService, languageService, circleDataService, stateDataService, districtDataService,
+                districtService);
+
+        Subscriber subscriber = subscriberDataService.create(new Subscriber(1000000000L));
+
+        subscriptionService.createSubscription(subscriber.getCallingNumber(), rh.hindiLanguage(),
+                sh.childPack(), SubscriptionOrigin.IVR);
+        subscriptionService.createSubscription(subscriber.getCallingNumber(), rh.hindiLanguage(),
+                sh.pregnancyPack(), SubscriptionOrigin.IVR);
+
+        transactionManager.commit(status);
+    }
+
+
+    // Test deactivation of specific msisdn
+    @Test
+    public void testDeactivateSpecificMsisdn() throws IOException, InterruptedException, URISyntaxException {
```

Comment:
Add few more tests if number is not valid returns 403.
Check if Hold / Active Subscription are deactivated
Check if Deactivation status got changed.